diff --git a/node_modules/gatsby-image/index.js b/node_modules/gatsby-image/index.js
index 4f90937..691efd6 100644
--- a/node_modules/gatsby-image/index.js
+++ b/node_modules/gatsby-image/index.js
@@ -286,13 +286,13 @@ var noscriptImg = function noscriptImg(props) {
   var title = props.title ? "title=\"" + props.title + "\" " : "";
   var alt = props.alt ? "alt=\"" + props.alt + "\" " : "alt=\"\" "; // required attribute
 
-  var width = props.width ? "width=\"" + props.width + "\" " : "";
-  var height = props.height ? "height=\"" + props.height + "\" " : "";
+  // var width = props.width ? "width=\"" + props.width + "\" " : "";
+  // var height = props.height ? "height=\"" + props.height + "\" " : "";
   var crossOrigin = props.crossOrigin ? "crossorigin=\"" + props.crossOrigin + "\" " : "";
   var loading = props.loading ? "loading=\"" + props.loading + "\" " : "";
   var draggable = props.draggable ? "draggable=\"" + props.draggable + "\" " : "";
   var sources = generateNoscriptSources(props.imageVariants);
-  return "<picture>" + sources + "<img " + loading + width + height + sizes + srcSet + src + alt + title + crossOrigin + draggable + "style=\"position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\"/></picture>";
+  return "<picture>" + sources + "<img " + loading + sizes + srcSet + src + alt + title + crossOrigin + draggable + "style=\"position:absolute;top:0;left:50%;opacity:1;object-fit:contain;object-position:center;transform:translate(-50%,0)\"/></picture>";
 }; // Earlier versions of gatsby-image during the 2.x cycle did not wrap
 // the `Img` component in a `picture` element. This maintains compatibility
 // until a breaking change can be introduced in the next major release
@@ -340,11 +340,12 @@ var Img = /*#__PURE__*/_react.default.forwardRef(function (props, ref) {
     style: (0, _extends2.default)({
       position: "absolute",
       top: 0,
-      left: 0,
-      width: "100%",
-      height: "100%",
-      objectFit: "cover",
-      objectPosition: "center"
+      left: "50%",
+      // width: "100%",
+      // height: "100%",
+      objectFit: "contain",
+      objectPosition: "center",
+      transform: "translate(-50%,0)",
     }, style)
   }));
 });
